ui/components/app/signature-request/signature-request-message/signature-request-message.js

```diff
@@ -81,7 +82,7 @@ export default function SignatureRequestMessage({
           marginLeft={4}
           className="signature-request-message__title"
         >
-          {t('signatureRequest1')}
+          {primaryType}
```

Comment:
May be `{primaryType || t('signatureRequest1')}`.

Reply:
Ignore this as I found every typed message except V1 has a primary type. we are not using this component for V1.